exercises/branching-conditionals/isValidTriangle/isValidTriangle.js

```diff
@@ -19,19 +19,43 @@ function isValidTriangle(a, b, c) {
     from 0º and you drawing the third side. What's the largest this third side
     could ever be?
   */
+ let maxLength = 0;
```

Comment:
@jazheng1 Nice! You see that if `a,b,c` were sorted then the question becomes easier. Of course, you probably _also_ realize that your code is much longer than a simple:

```javascript
return a + b > c && a + c > b && b + c > a;
```

Buuuuut, you can use [Array.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) to sort the array!

```javascript
function isValidTriangle(a, b, c) {
  let sorted = [a, b, c].sort();

  return sorted[0] + sorted[1] > sorted[2];
}
```

You can also use something called [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to make it even more concise:

```javascript
function isValidTriangle(a, b, c) {
  let [x, y, z] = [a, b, c].sort();

  return x + y > z;
}
```